code/modules/client/preference_setup/loadout/lists/belt.dm

```diff
@@ -10,7 +10,7 @@
 	path = /obj/item/storage/belt/utility
```

Comment:
please replace the general belt with
/obj/item/storage/belt/champion
i didnt notice before hand that you were adding in a item not really meant for use >~< my bad